### ISSUE TITLE
file detail don't look up

### ISSUE THREAD

**Issue by LengYi** [nst/RuntimeBrowser#24]
xxx.h file detail don't look up,

**Comment by MemoryReload** [comment]
@LengYi   Yeah, I also suffered this problem. tap the .h file, the cell just become selected and never show the  method details. I've red the source code, it same that, the cell selected delegate method is not hooked and some code is commented out.

**Comment by MemoryReload** [comment]
@LengYi   I looked the old issues and found the reason why this happened. Close this issue and see issue #6 . : )
